server/controllers/visitation.controllers.js

```diff
@@ -3,32 +3,38 @@ import EMAIL_CONTENT from '../../mailer';
 import { sendMail } from '../services/mailer.service';
 import httpStatus from '../helpers/httpStatus';
 
-const BALLERS_EMAIL = process.env.BALLERS_EMAIL || 'dev@highrachy.com';
-
 const VisitationController = {
   book(req, res, next) {
     const booking = req.locals;
     const { user } = req;
     scheduleVisitation({ ...booking, userId: user._id })
-      .then((schedule) => {
+      .then(({ schedule, vendorEmail }) => {
         const contentBottom = `
           <strong> Name: </strong>: ${schedule.visitorName}<br /> 
           <strong> Phone: </strong> ${schedule.visitorPhone}<br />
           <strong> Email: </strong> ${
             schedule.visitorEmail ? schedule.visitorEmail : 'No email provided'
           } <br />
         `;
-        sendMail(EMAIL_CONTENT.SCHEDULE_VISIT, { email: BALLERS_EMAIL }, { contentBottom });
+        sendMail(EMAIL_CONTENT.SCHEDULE_VISIT, { email: vendorEmail }, { contentBottom });
```

Comment:
Nice thinking here. However, we want the mail to also retrieve the company name, so we might need to add the vendor company name here so that it can work as expected. 

https://github.com/Highrachy/ballers-api/blob/76129cd26df03c27bc2518302c3965ecd7de6cbb/server/services/mailer.service.js#L64

I didn't test it though, you can start by passing the whole vendor Object where vendor stands for all the user information.

```suggestion
        sendMail(EMAIL_CONTENT.SCHEDULE_VISIT, vendor, { contentBottom });
```